<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>908</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="img_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLCDNumber" name="param2_lcd">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="param2_label">
              <property name="text">
               <string>param2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="param2_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSpinBox" name="param2_spin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLCDNumber" name="minrad_lcd">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="minrad_label">
              <property name="text">
               <string>min rad</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="minrad_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLCDNumber" name="maxrad_lcd">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="maxrad_label">
              <property name="text">
               <string>max rad</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="maxrad_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLCDNumber" name="dp_lcd">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>dp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="dp_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLCDNumber" name="mindist_lcd">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="mindist_label">
              <property name="text">
               <string>minDist</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="mindis_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSpinBox" name="mindist_spin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLCDNumber" name="param1_lcd">
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="param1_label">
              <property name="text">
               <string>param1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="param1_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSpinBox" name="param1_spin"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dp_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dp_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mindis_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mindist_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>param1_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>param2_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minrad_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>minrad_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxrad_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>maxrad_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mindis_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_min_dist(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>863</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dp_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_dp(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_param1(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>param2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_param2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>761</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minrad_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_min_rad(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>812</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxrad_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update_max_rad(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>863</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_method(int)</slot>
  <slot>update_dp(int)</slot>
  <slot>update_min_dist(int)</slot>
  <slot>update_param1(int)</slot>
  <slot>update_param2(int)</slot>
  <slot>update_min_rad(int)</slot>
  <slot>update_max_rad(int)</slot>
 </slots>
</ui>
